docs/tables/aws_alb_access_log/queries.md

```diff
@@ -0,0 +1,347 @@
+  elb,
+  tp_index as account_id,
+  client_ip,
+  target_ip,
+  request,
+  elb_status_code,
+  target_status_code,
+  error_reason
+from
+  aws_alb_access_log
+where
+  target_status_code is null
+  and elb_status_code = 502
+order by
+  timestamp desc;
+```
+
+### SSL cipher vulnerabilities
+
+Detect usage of deprecated or insecure SSL ciphers.
+
+```sql
+select
+  ssl_cipher,
+  ssl_protocol,
+  count(*) as request_count
+from
+  aws_alb_access_log
+where
+  ssl_protocol in ('TLSv1.1', 'TLSv1', 'SSLv3', 'SSLv2') -- Insecure protocols (TLSv1.1, TLSv1, SSLv3, SSLv2)
+group by
+  ssl_cipher,
+  ssl_protocol
+order by
+  request_count desc;
+```
+
+### Suspicious user agents
+
+Identify potentially suspicious user agents making requests.
+
+```sql
+select
+  user_agent,
+  count(*) as request_count
+from
+  aws_alb_access_log
+where
+  user_agent like '%bot%'
+  or user_agent like '%curl%'
+  or user_agent like '%wget%'
+group by
+  user_agent
+having
+  count(*) > 100
+order by
+  request_count desc;
+```
+
+## Operational Examples
+
+### Slow response times
+
+Top 10 requests with unusually high processing times.
+
+```sql
+select
+  timestamp,
+  elb,
+  tp_index as account_id,
+  request,
+  client_ip,
+  target_ip,
+  request_processing_time,  -- Time taken by load balancer to process request
+  target_processing_time,   -- Time taken by target to process request
+  response_processing_time, -- Time taken to process response
+  (request_processing_time + target_processing_time + response_processing_time) as total_time
+from
+  aws_alb_access_log
+where
+  (request_processing_time + target_processing_time + response_processing_time) > 1 -- Requests taking longer than 1 second
+order by
+  total_time desc
+limit 10;
+```
+
+### Target health issues
+
+Identify targets that are returning a high number of errors.
+
+```sql
+select
+  target_ip,
+  target_status_code,
+  count(*) as error_count
+from
+  aws_alb_access_log
+where
+  target_status_code >= 400
+group by
+  target_ip,
+  target_status_code
+having
+  count(*) > 100
+order by
+  error_count desc;
+```
+
+## Volume Examples
+
+### High traffic periods
+
+Detect periods of unusually high request volume.
+
+```sql
+select
+  date_trunc('minute', timestamp) as request_minute,
+  elb,
+  count(*) as request_count
+from
+  aws_alb_access_log
+group by
+  request_minute,
+  elb
+having
+  count(*) > 1000
+order by
+  request_count desc;
+```
+
+### Large response sizes
+
+Track requests generating unusually large responses.
+
+```sql
+select
+  timestamp,
+  elb,
+  tp_index as account_id,
+  request,
+  client_ip,
+  sent_bytes,
+  received_bytes
+from
+  aws_alb_access_log
+where
+  sent_bytes > 10485760 -- 10MB
+order by
+  sent_bytes desc;
+```
+
+### Count requests by Listener Type (HTTP, HTTPS, HTTP/2, WebSockets, etc.)
+
+Identify how many requests were received for each listener type.
+
+```sql
+select
+  listener_type,
+  count(*) as request_count
+from (
+  select
+    case
```

Comment:
Can we also get this info from the `type` column? Seems like it would be easier based on the info in https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html#access-log-entry-format